docs/reference/components/splunk.cue

```diff
@@ -11,13 +11,13 @@ components: _splunk: {
 			default_namespace: "vector"
 			tags:              telemetry.metrics._component_tags
 		}
-		source_missing_keys_total: {
+		source_template_rendering_errors_total: {
```

Comment:
I'm not sure what is this used for, it is not referenced any where. What should I do with this?

Reply:
👍 this seems to have been an accidental inclusion. Could you drop it and `sourcetype_template_rendering_errors_total`?

Reply:
Removed them!